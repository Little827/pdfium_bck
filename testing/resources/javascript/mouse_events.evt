# Open the dropdown to make sure it displays correctly.
# Mouse in, mouse out.
mousemove,125,225
mousemove,125,100
# Mouse in, click in field.
mousemove,125,225
mousedown,left,125,225
mouseup,left,125,225
# Mouse out, click elsewhere.
mousemove,125,100
mousedown,left,125,100
mouseup,left,125,100
# Mouse in, focus.
mousemove,125,225
focus,125,225
# Mouse out, unfocus.
mousemove,125,100
focus,125,100
