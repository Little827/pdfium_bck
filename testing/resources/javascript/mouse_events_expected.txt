Alert: enter
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
Alert: exit
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
Alert: enter
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
Alert: down
Doc Submit Form: url=https://example.com + 174 data bytes: %FDF-1.2
1 0 obj
<</FDF<</F<</F(myfile.pdf)/Type/Filespec/UF(myfile.pdf)>>/Fields[<</T(name)/V(Tralfaz)>><</T(age)/V(12)>>]>>>>
endobj

trailer
<</Root 1 0 R>>
%%EOF

Doc Print: 1, 0, 0, 0, 0, 0, 0, 0
Doc Print: 0, 1, 10, 1, 1, 1, 1, 1
Doc Print: 1, 0, 0, 0, 0, 0, 0, 0
Doc Print: 1, 42, 17, 1, 1, 1, 1, 1
Alert: focus
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
Alert: up
Doc Submit Form: url=https://example.com + 19 data bytes: name=Tralfaz&age=12
Doc Print: 1, 0, 0, 0, 0, 0, 0, 0
Alert: exit
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
Alert: enter
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
Alert: focus
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
Alert: exit
Alert: PASS: this.submitForm blocked with Document.submitForm: User gesture required.
Alert: PASS: this.print blocked with Document.print: User gesture required.
